README.md

```diff
@@ -19,7 +19,7 @@ To get a local copy up and running follow these simple exampl
 
 #### Live demo
 
-[Math Magicians Website](https://rid-bookstore-cms.netlify.app/)
+[BookStore CMS Website](https://rid-bookstore-cms.netlify.app/)
 
 ### **Option 2**
 
```

Comment:
- [ ] **Optional:**
https://github.com/ridwanediallo/Bookstore-CMS/blob/f4fa6fab15e79ab8ebd213669343b22ed10de63c/src/components/BookList.js#L10-L12
- Kindly remove commented code. It is unprofessional.

Reply:
Okay, @brytebee thank you 🙏